IviDriverCore/1.0/Spec/IviDriverCore.md

```diff
@@ -0,0 +1,770 @@
+
+- **Known Issues:**
+  The Known Issues item shall specify a list of known issues that reflect all test failures that were not fixed. If there are no known issues, indicate None.
+
+#### Additional Compliance Information Section
+
+The Additional Compliance Information section informs the user about additional information that relates to compliance but is not included in other defined items. This section is not present if no additional compliance information exists.
+
+#### Alternate Compliance Documentation Formats
+
+Drivers that otherwise comply with the *IVI Driver Core Specification* may use the format described in *IVI-3.1 Architecture Specification*.  That format shall be an acceptable alternate format for drivers that conform to that specification.
+
+### README.md Documentation
+
+The driver README.md file includes details that will help users get started with the driver.  It shall contain:
+
+- The location of the `___Introduction to <DriverIdentifier>___.<DocumentExtension>` documentation file.
+- The location of the help documentation file(s).
+- The text of the compliance documentation. The compliance documentation includes items normally associated with a `README.md` file, such as driver version information, known issues, and driver dependencies. The `README.md` file may also contain the location of a separate compliance document that also contains the compliance documentation.
+
+The `README.md` file may contain additional information that users may need to know, or find useful, before using the driver.
+
+## File Versioning
+
+IVI driver Windows DLLs shall contain a Windows version resource with the three entries: `CompanyName`, `ProductName`, and `FileVersion`.
+
+- `CompanyName` is a string that contains the value of the Driver Vendor or Driver Vendor property that the IVI driver returns.
+
+- `ProductName` is a string that includes the value of the Driver Identifier property that the IVI driver returns.
+
+- `FileVersion` is a string in the following format:
+  
+  `MajorVersion.MinorVersion.BuildVersion[.InternalVersion]`
+
+`MajorVersion`, `MinorVersion`, `BuildVersion`, and `InternalVersion` are decimal numbers greater than or equal to zero and less than or equal to 65535. The MajorVersion shall not be zero, except for a pre-release version of the initial release of a driver. Each number may contain leading zeros but may not exceed 5 digits. The number fields are separated by periods, with no embedded spaces. The maximum valid FileVersion is 65535.65535.65535.65535.
+
+A FileVersion string shall contain at least a `MajorVersion`, `MinorVersion`, and `BuildVersion`. A driver shall be released with an incremented `MajorVersion` or `MinorVersion` number when any of the following conditions are true:
+
+- The new version of the driver contains changes in its API syntax.
```

Comment:
I think we should make a distinction or at least call out breaking changes vs non-breaking changes.

For example
* Adding a new method/property is not a breaking change but it is a change in API syntax.
* Renaming a method/property or changing the arguments to a method _is_ a breaking change in API syntax.

Which one is it called out here?
Should we explictly disallow breaking changes, or force a major version bump in the case of a breaking change?